cubano-http/src/main/java/org/concordion/cubano/driver/http/HttpDownloader.java

```diff
@@ -3,6 +3,19 @@
 import java.io.File;
 import java.io.IOException;
 
+/**
+ * File download interface.
+ */
 public interface HttpDownloader {
+
+    /**
+     * Download file.

```

Comment:
I know this might be part of the coding standard, but it really doesn't add anything. My preference would be to change the coding standard and only apply meaningful comments.

Reply:
Agreed, I've added this to my list of things to discuss with you next time we catch up...